chapters/1-the-box/index.html

```diff
@@ -210,10 +210,11 @@ <h3 class="magic">Flexible inputs</h3>
 
                 <p>If you want <code>height</code> and <code>width</code> to behave in the most intuitive way, <a href="http://www.paulirish.com/2012/box-sizing-border-box-ftw/">listen to Paul Irish</a><sup><a href="#cite-3">[3]</a></sup> and put this at the top of your CSS:</p>
 
-                <pre><code>*, *:before, *:after {
-    -webkit-box-sizing: border-box; /* Needed for mobile WebKit */
-    -moz-box-sizing: border-box; /* Needed for Firefox */
-    box-sizing: border-box
+                <pre><code>html {
+  box-sizing: border-box;
```

Comment:
Sorry, please remove these `;` as well as that is the way the rest of MoCSS is done